core/templates/dev/head/forms/form_builder_templates.html

```diff
@@ -132,7 +132,7 @@
 <script type="text/ng-template" id="schemaBasedEditor/float">
   <span ng-model="localValue" ui-validate="'validate($value)'">
     <span ng-form="floatForm">
-      <span ng-if="!expressionMode">
+      <span ng-if="!expressionMode && uiConfig().slider === undefined">
```

Comment:
Yeah that caused problems when slider is not defined in other places where the float schema is used.